### ISSUE TITLE
Just installed a wallet on Chrome but get this error : Unexpected token '<' , "<!DOCTYPE "... is not valid JSON

### ISSUE THREAD

**Issue by djrces** [unisat-wallet/libbrc20-indexer#8]
Any idea why?
I sent some BTC but cannot see the balance even tough it's on the blockchain explorer :-(

**Comment by knuxel** [comment]
Same here. Cannot make any transactions.

**Comment by RSHills** [comment]
Got back that a block reorganisation on the Bitcoin network at block height 792.379 occurred a moment ago, so Unisat is down but are fixing it. Will wait a bit.

**Comment by babyneo1** [comment]
yeah i just had the same , marketplace on unisat is down but i still managed to inscribe a .sat then after that had the same wallet issue

**Comment by RSHills** [comment]
thanks

**Comment by babyneo1** [comment]
hopefully sorts it self out i tried uninstalling wallet and adding it again and restoring. same issue atm :)

**Comment by RSHills** [comment]
Same here, tried that as well. Gives us some time to think of some new Ordinals to inscribe once solved. : )

**Comment by babyneo1** [comment]
wallet does open on unisat webite still tho, but limited use . haha just did a couple of good ones but couldnt 100% if i was first to it . hope so . took a good youtube channel one if it is $$$ hahah

**Comment by Arobs598** [comment]
I have the same problem, any ideas?

**Comment by babyneo1** [comment]
seems like wait it out not had issue before

**Comment by sattoboi** [comment]
i have the same problem, in two new wallets i received .0058 btc and it has that error "Unexpected token '<' , "<!DOCTYPE "... is not valid JSON".  and doesn't show a btc balance in the Unisat Chrome extension. if i open the history for the wallet on mempool it shows my balance that I received so I'm hoping like yall it gets fixed soon on the actual wallet

**Comment by Arobs598** [comment]
Yeah I can still see my ordi when I look at ordinals wallet .com but unisat still not responding. When I pull it up in the browser it shows code 502.

**Comment by sturletnapol** [comment]
I think, It's error on unisat server.
We need the help of Unisat managers.

**Comment by gvlrich** [comment]
I just got this error. Looking for some info. I was going to reinstall but I see it is not working.

**Comment by gvlrich** [comment]
I just joined the discord and saw this. 
"Immune — Today at 12:07 PM
@everyone
Today we witnessed a block reorganisation on the Bitcoin network at block height 792,379. As a security precaution we have temporarily closed UniSat Market. UniSat Wallet is temporarily unavailable because our indexer needs to catch up with the correct chaintip.

There is no need for panic, we are not affected by this reorg, your assets are safe. We are actively monitoring the situation and the market will be recovered in the next few hours.

There is no need for panic, we are not affected by this reorg, your assets are safe. We are actively monitoring the situation and the market will be recovered in the next few hours."

**Comment by sturletnapol** [comment]
It solved!

**Comment by sattoboi** [comment]
problem solved! glad they took care of it btc is available and wallet it gud

**Comment by photojacketman** [comment]
I got this problem today, seems there might be another issue potentially on their side. Please update!
